services/monitoring/template.env

```diff
@@ -1,5 +1,6 @@
 MONITORING_DOMAIN=${MONITORING_DOMAIN}
 MONITORING_PROMETHEUS_RETENTION=${MONITORING_PROMETHEUS_RETENTION}
+MONITORING_PROMETHEUS_FEDERATION_RETENTION=${MONITORING_PROMETHEUS_FEDERATION_RETENTION}
```

Comment:
I can't find a RR adding this ENV to `repo.config`. I assume it will follow this MR.

Reply:
Good catch thank you, I in fact forgot about this. I will add it now